### DISCUSSION TITLE
멘토님 main브렌치에  수정 요청드립니다.

### DISCUSSION THREAD

**Discussion by yoonucho** [ThinkAboutSoftware/AcademicConference#18] [General]
![image](https://user-images.githubusercontent.com/2981954/115102548-87571f80-9f86-11eb-8919-49e1f4425980.png)

저만 100년을 앞서있군요 ㅋㅋ ㅋㅋㅋ

**Comment by jongfeel** [comment]
> ![image](https://user-images.githubusercontent.com/2981954/115102548-87571f80-9f86-11eb-8919-49e1f4425980.png)
> 
> 
> 
> 저만 100년을 앞서있군요 ㅋㅋ ㅋㅋㅋ

미래에서 오셨군요 다음주 로또 번호좀 알려주세요 ㅋㅋㅋㅋㅋㅋㅋ

**Reply by yoonucho** [reply]
제가 뭔 짓을 한걸까요? ㅋ

**Reply by jongfeel** [reply]
아직 branch 삭제 안되어 있으니까 
폴더 이름 변경에 대한 pull reqeust를 주는게 더 좋아 보여요.

**Reply by yoonucho** [reply]
넵 그러면 퇴근후에  pr드릴게요!

**Reply by yoonucho** [reply]
아이공 어제 야근하고 늦게와서 이제 pr드렸습니다! ㅠ

**Reply by Joseph-Cha** [reply]
ㅋㅋㅋㅋㅋㅋㅋ
